### ISSUE TITLE
import oemof packages in the same namespace

### ISSUE THREAD

**Issue by uvchik** [oemof/oemof#78]
I had problems importing the two packages `oemof` and `oemof.db` in the same namespace. I fixed it by adding the following line in the top-level `__init__.py` of `oemof_db.`

``` python
from pkgutil import extend_path
__path__ = extend_path(__path__, __name__)
```

@gnn : Please check if it is the right solution but it fixes all import errors for me. I added it to the dev-branch of `oemof.db`.

**Comment by uvchik** [comment]
I had to add these lines to the oemof package, too, to get it work at home (2a9eda5c3). This is explained here: http://stackoverflow.com/questions/1675734/how-do-i-create-a-namespace-package-in-python

See also here: https://docs.python.org/3/library/pkgutil.html

I hope everything works fine now. It could effect issue #79.

@gnn : Please check if I did it right and close this issue.

**Comment by cswh** [comment]
@gnn are you still on that?

**Comment by gnn** [comment]
Yep. I'm still on that and I can actually create installations which still breaks. Testing all the variations is a bit time consuming, that's why I wasn't able to fully fix it yet. But I hope to get this resolved for the May release.

**Comment by gnn** [comment]
I did some more testing and it worked, regardless of the order of installation of `oemof.db` and `oemof`. Ulf ran into trouble though as it didn't work for him. I'm moving this bug to the next milestone, knowing that it is known to work sometimes (most of the time?) and that right now the hard part is figuring out when breakage occurs.

**Comment by gnn** [comment]
It's still not entirely clear when this breaks. I really want to pin this down so I'm moving it to the next release.

**Comment by gnn** [comment]
Closed in favour of #269.
